main.py

```diff
@@ -1220,7 +1272,6 @@ def get_inspections(self) -> None:
             value = self.inspections_sheet.cell(row=self.inspections_count, column=1).value
             if not value:
                 break
```

Comment:
```suggestion
                break

```
なんかミスって改行まで消してしまった...